### PR TITLE
Added OWNERS file in install_yamls

### PR DESCRIPTION
**Pull request by amolkahat**
Added OWNERS file

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (sandeepyadav93) [inline OWNERS:1]: Remove these comments..
Please see examples here: https://github.com/openstack-k8s-operators/keystone-operator/blob/master/OWNERS

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - stuggi
+  - viroel
+  - gibizer
+  - fultonj
+  - fmount
+  - slagle
+  - karelyatin
+  - fao89
+  - raukadah
+options: {}
```
Comment (sandeepyadav93) [inline OWNERS:21]: not needed

**Review by sandeepyadav93 — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 2 above)
Comment (amolkahat) [reply OWNERS:21]: Done.

File: OWNERS (hunk 1 above)
Comment (amolkahat) [reply OWNERS:1]: Done.

**Review by abays — APPROVED** [review]
/lgtm

**Review by fmount — APPROVED** [review]

**Pull request merged**